### ISSUE TITLE
add write support

### ISSUE THREAD

**Issue by gesellix** [gesellix/keepass-node#2]
this feature is being implemented in the [write-support branch](https://github.com/gesellix/keepass-node/tree/write-support).

use cases:
- [x] add a group/subgroup to the selected group
- [x] add an entry to the selected group
- [ ] modify an existing entry
- [ ] modify an existing group
- [ ] sort groups
- [ ] sort entries

probably necessary:
- [x] authentication, preferrably using jwt. see https://github.com/auth0/angular-jwt and https://github.com/auth0/express-jwt and the blog post at https://auth0.com/blog/2014/10/01/handling-jwts-on-angular-is-finally-easier/ for possible solutions.

**Comment by sercheo87** [comment]
Reading key file , now exercise with data base keepass version 2 because not decrypted.

**Comment by gesellix** [comment]
@sercheo87 can you rephrase, please? I don't perfectly understand what you meant. Would you like key file support to be added? What do you mean with "... because not decrypted."?
